src/lexer.rs

```diff
@@ -160,6 +101,7 @@ fn granularize(block: &str) -> Result<Vec<Token>> {
             ")" => CloseRound,
             "?" => Question,
             "-" => Dash,
+            "=" => Assignment,
```

Comment:
You will want to add the assigment symbol to the end of [this regex](https://github.com/cobalt-org/liquid-rust/blob/master/src/lexer.rs#L121) so that it gets split properly.

Feel free to also add a test below if you like, I'll take a stab at refactoring these tests soon anyway I think.